template.rb

```diff
@@ -177,6 +177,9 @@ def render_file(filename)
 
           config.web_console.permissions = whitelisted_ips
           config.web_console.whiny_requests = false
+
+          BetterErrors::Middleware.allow_ip! '10.138.0.0/16'
```

Comment:
@jelaniwoods This will, effectively, allow _all_ visitors (from any external IP address) to access the consoles, correct?

In that case, is there a way to just explicitly tell the `better_errors` and `web-console` gems to not take the security precaution at all?

I'm worried some day that Gitpod will change their internal subnet address to something other than `10.138...` and then students will be stuck. At a minimum, we should remove `console_ip_whitelist` so that they will just fall back to the dumb error page in that case — right?

Reply:
@raghubetina yes `console_ip_whitelist` should be removed so we default back to the regular error page, I forgot to update the description.

I'll look more into how `better_errors` works in a fork, since it didn't seem possible to disable security measures.